database.sql

```diff
@@ -1,23 +1,100 @@
--- Step 1: Insert a user
+
+
+
+
+
+CREATE TABLE IF NOT EXISTS Users(
+    UserID TEXT UNIQUE NOT NULL,
+    password CHAR(80) NOT NULL,
+    firstName CHAR(80) NOT NULL,
+    lastName CHAR(80) NOT NULL,
+    PRIMARY KEY(USERID)
+    
+);
+
+
+CREATE TABLE IF NOT EXISTS Orders (
+    OrderID TEXT UNIQUE NOT NULL,
+    UserID TEXT NOT NULL,
+    BusinessID TEXT NOT NULL,
+    ShopperID TEXT NOT NULL,
+    Status TEXT,
+    fulfillmentTime INT NOT NULL,
+    addressprimary VARCHAR(255) NOT NULL, 
+    addresssecondary VARCHAR(255) NOT NULL,
+    city VARCHAR(100) NOT NULL,
+    state VARCHAR(50 NOT NULL),
+    zip_code VARCHAR(20) NOT NULL
+    PRIMARY KEY(OrderID),
+    FOREIGN KEY (UserID) REFERENCES Users(UserID),
+    FOREIGN KEY (BusinessID) REFERENCES BusinessInfo(BusinessID),
+    FOREIGN KEY (ShopperID) REFERENCES JobBoard(ShopperID),
+    FOREIGN KEY (fulfillmenttime) REFERENCES JobBoard(fulfillmenttime)
+
+
+);
+
+
+
+CREATE TABLE IF NOT EXISTS BusinessInfo(
+    BusinessID TEXT UNIQUE NOT NULL,
+    addressprimary VARCHAR(255) NOT NULL, 
+    addresssecondary VARCHAR(255),
+    city VARCHAR(100) NOT NULL,
+    state VARCHAR(50 NOT NULL),
+    zip_code VARCHAR(20) NOT NULL,
+    PRIMARY KEY (BusinessID)
+
+
+);
+
+
+CREATE TABLE IF NOT EXISTS OrderItems(
+    OrderID TEXT UNIQUE NOT NULL,
+    UPC TEXT UNIQUE NOT NULL,
+    quantity INT NOT NULL,
+    PRIMARY KEY (OrderID, UPC),
+    FOREIGN KEY (OrderID) REFERENCES Orders(OrderID),
+    FOREIGN KEY (UPC) REFERENCES Items(UPC)
+
+
+);
+
+CREATE TABLE IF NOT EXISTS JobBoard(
+    ShopperID TEXT UNIQUE NOT NULL,
+    fulfillmenttime TEXT UNIQUE NOT NULL,
```

Comment:
Do you mind going through and making consistent variable capitalization? The backend currently assumes camelCase for all names

ex: "addressPrimary," "fulfillmentTime," "state," "city"